src/views/index.js

```diff
@@ -0,0 +1,29 @@
+import 'babel-polyfill';
```

Comment:
Should this file be within the views folder? We have one already in the source folder